setup.cfg

```diff
@@ -79,6 +85,18 @@ filterwarnings =
     ignore:numpy\.ufunc size changed:RuntimeWarning
     ignore:numpy\.ndarray size changed:RuntimeWarning
 
+# The following is to work around an issue building the documentation (see
+# #383).  The prefered approach is to use doctest-requires at the code-block
+# level (e.g., this marks all orbit.rst tests as needed oorb, but only 2 of 9
+# blocks require it).  Relevant doctest-requires directives have been commented
+# out.
+doctest_subpackage_requires =
+    docs/sbpy/photometry.rst = astroquery
+    docs/sbpy/data/ephem.rst = astroquery
+    docs/sbpy/data/obs.rst = astroquery
+    docs/sbpy/data/orbit.rst = astroquery,oorb
+    docs/sbpy/data/phys.rst = astroquery
+
```

Comment:
Not sure how I should review this block.

1. Why do we only specify astroquery here, not others such as scipy?
2. Do you want me to go through all the document files to see if there is anything missing here?

Reply:
This is specifically for the work-around documented in the comments.  If you wanted to verify that this is needed, then comment out any of lines 94 to 98 and run the tests.